frontend/src/components/about/Platform.vue

```diff
@@ -49,13 +49,20 @@
       </p>
 
       <h3 id="citation" class="title is-3 pt-6">Citation</h3>
-
-      <h4 class="title is-4">Latest publication</h4>
       <p>
         If you use Metabolic Atlas in your work, or any of the animal models, please cite our latest
         publication. In addition, if you use Human-GEM, please also cite the corresponding
         publication.
       </p>
+      <h4 id="citation-v3" class="title is-4">3.0</h4>
```

Comment:
I would suggest having title `Version 3.0` here and similarly in the other version titles.

Reply:
➕

Reply:
I also agree. Changed in 8afdc46